learners/trial-runs.md

```diff
@@ -2,5 +2,124 @@
+and outlined a narrative to lead learners through that chapter.
+You should use the trial run as an opportunity to teach this chapter of your lesson to a real audience.
+Note that it will probably be helpful to spend some time 
+writing some of the supporting information for your lesson for this trial run.
+For example, it is likely that some setup instructions will need to be provided in
+the `learners/setup.md` file.
+
+### For groups of collaborators
+
+During the training, you should have been working on consecutive chapters in your lesson.
+Use the trail run as an opportunity for you to test the lesson content together,
+with each trainee teaching the chapter they have been working on during the training 
+in a single trial run event.
+This will give you an opportunity to observe each other teaching,
+to help each other gather feedback and information to help improve the lesson (see below),
+and to identify any gaps or overlaps that become apparent when moving from one
+chapter to the next.
+
+### For Lesson Developers working alone
+
+We ask you to collect a lot of information during the trial run (see below),
+and it will be difficult to do this while also devoting yourself to delivering the lesson.
+For that reason, we recommend that you try to find someone who can attend your trial run
+as an observer, to take notes and give you feedback from their perspective.
+Ideally, this would be someone who you work with and who is already familiar 
+with the topic that will be taught in your lesson chapter.
+That will make it easier for them to provide their own insights, 
+and to properly capture details in the notes they take during the trial run.
+If you are unable to recruit a colleague or peer to do this for your trial run,
+please discuss it with your trainers and/or send an email to
+[curriculum@carpentries.org](mailto:curriculum@carpentries.org) to request support.
+
+
+## What information should be collected from the Trial Run?
+
+**During the trial run**, it will be helpful to take notes to answer the following questions:
+
+- How long did it take to teach the chapter?
+- How long did learners need to complete each exercise?
+- What questions did learners have during the trial run?
+- What technical difficulties were encountered during the trial run?
+- What parts of the lesson appeared to be confusing for learners?
+
+**At the end of the trial run**, make sure to capture some feedback from the learners.
+You can use the methods described in [Preparing to teach](../episodes/18-preparing.md) to
+gather feedback, e.g.
+
+- Minute cards
+- One up, one down
+- Ask learners about the exercises in your chapter
+
+**As soon as possible after the trial run** has finished -
+ideally immediately afterwards -
+take some time to reflect on the following questions,
+based on the feedback collected and from your own perspective 
+as the instructor and author of the lesson content,
+Make some notes on your answers to these questions.
```

Comment:
```suggestion
as the instructor and author of the lesson content, and make some notes on your answers to these questions.
```
Think it could also be two sentences with a period after "content" instead of a comma.